server/index.js

```diff
@@ -32,6 +33,7 @@ app.all('*', (req, res) =>
 );
 
 app.use(ErrorHandler.sendError);
+app.use(Authenticator.deleteReservedIfExpired);
```

Comment:
When does this run? Everytime?

Reply:
Kinda, It should always check for it.

Reply:
I think it is only necessary when before reserving or lending a book